package/cloud-regionsrv-client.changes

```diff
@@ -1,3 +1,29 @@
+-------------------------------------------------------------------
+Wed Sep 17 07:47:42 UTC 2025 - Marcus Schäfer <marcus.schaefer@suse.com>
+
+- Update to version 10.5.3
```

Comment:
Due to the spec file issue this is actually not releasable. We still have the open conversation about how we want to handle version bumps in this project. I am generally not in favor that every change constitutes a version bump, but can certainly be convinced otherwise.

Reply:
I bumped the patch version due to changes on the logging which results in a more verbose information of the log file. Thus this refactor had an impact for users which imho requires a patch level bump.

> I am generally not in favor that every change constitutes a version bump

I agree and I only bump versions on changes or changesets that impacts behavior or fix a bug. I don't see what should be wrong with that